src/main/java/com/yahoo/bullet/kafka/KafkaConfig.java

```diff
@@ -51,15 +51,21 @@
     public static final String RESPONSE_PARTITIONS = KAFKA_NAMESPACE + "response.partitions";
     public static final String REQUEST_TOPIC_NAME = KAFKA_NAMESPACE + "request.topic.name";
     public static final String RESPONSE_TOPIC_NAME = KAFKA_NAMESPACE + "response.topic.name";
+    public static final String PARTITION_ROUTING_DISABLE = KAFKA_NAMESPACE + "partition.routing.disable";
 
     // Kafka PubSub Subscriber properties
     public static final String MAX_UNCOMMITTED_MESSAGES = KAFKA_NAMESPACE + "subscriber.max.uncommitted.messages";
+    public static final String RATE_LIMIT_ENABLE = KAFKA_NAMESPACE + "subscriber.rate.limit.enable";
+    public static final String RATE_LIMIT_MAX_MESSAGES = KAFKA_NAMESPACE + "subscriber.rate.limit.max.messages";
+    public static final String RATE_LIMIT_INTERVAL_MS = KAFKA_NAMESPACE + "subscriber.rate.limit.interval.ms";
 
     // Defaults
     private static String TRUE = "true";
     private static String FALSE = "false";
     public static final String DEFAULT_KAFKA_CONFIGURATION = "bullet_kafka_defaults.yaml";
     public static final String DEFAULT_ENABLE_AUTO_COMMIT = TRUE;
+    public static final boolean DEFAULT_PARTITION_ROUTING_DISABLE = false;
```

Comment:
Can we invert this and call it PARTITION ROUTING ENABLE and default it to true instead? It's easier to think about that way, at least for me.